config/pod.yaml.hbs

```diff
@@ -128,6 +128,16 @@ spec:
     - mountPath: /opt/sdeventcache
       name: sd-event-cache
     {{/if}}
+  env:
+    {{#if secret_entity}}
```

Comment:
Is the intention here only mounts the `volume` if it has `secrets`?

Reply:
if this is a flag with `true` or `false` value, please rename it accordingly.